src/ui/content-settings/__tests__/ContentSettingsGeneralContainer.test.js

```diff
@@ -0,0 +1,35 @@
+import { mapStateToProps, mapDispatchToProps } from 'ui/content-settings/ContentSettingsGeneralContainer';
+import { CONTENT_SETTINGS_OK as C_OK } from 'testutils/mocks/contentSettings';
+
+const state = {
+  loading: {},
+  apps: {
+    cms: {
+      contentSettings: C_OK,
+    },
+  },
+};
+
+describe('content-settings/ContentSettingsGeneralContainer', () => {
+  it('maps stateprops property', () => {
+    const props = mapStateToProps(state);
```

Comment:
It seems to be lacking 2 more props to be asserted.
```suggestion
    const props = mapStateToProps(state);
    expect(props).toHaveProperty('indexesLastReloadDate');
    expect(props).toHaveProperty('indexesLastReloadResult');
```

Reply:
committed @raxkaynan